app/job.go

```diff
@@ -80,7 +82,32 @@ func (a *App) SessionHasPermissionToCreateJob(session model.Session, job *model.
 		model.JOB_TYPE_EXPORT_DELETE,
 		model.JOB_TYPE_CLOUD:
 		return a.SessionHasPermissionTo(session, model.PERMISSION_MANAGE_JOBS), model.PERMISSION_MANAGE_JOBS
+	}
 
+	return false, nil
+}
+
+func (a *App) SessionHasPermissionToReadJob(session model.Session, job *model.Job) (bool, *model.Permission) {
```

Comment:
Where is this function used?

Reply:
Thanks, I lost the changes for this. Not sure what happened but added now.